src/app/(sidebar)/endpoints/[[...pages]]/page.tsx

```diff
@@ -902,6 +910,14 @@ export default function Endpoints() {
       <Card>
         <form className="PageBody" onSubmit={handleSubmit}>
           {renderEndpointUrl()}
+
+          {/* display a missing request url banner if requestUrl is empty */}
+          {!requestUrl ? (
+            <Banner icon={<Icon.AlertTriangle />} variant="warning">
+              Set a {endpointNetwork.label} RPC URL in order to submit
+            </Banner>
```

Comment:
We should use `Notification` or `Alert` for inline messages. `Banner` should be used only on the top level (top or bottom of the page) and not inline (for example, the network reset message).

`Notification` and `Alert` don't have a yellow background. Ideally, we shouldn't customize SDS components and use them as is. But we can check with Design how important it is to make this background yellow and update it locally in CSS if needed.

Reply:
could we add yellow bg to `notification` and `alert` components in SDS instead of locally? i think it's important in this case to have the yellow. we'll also need this for other cases as well.

Reply:
Yes, we can update SDS to add the filled background option. :+1:

Reply:
@sdfcharles, are you sure you want a background for the `warning` variant on `Alert`? None of the `Alert` variants including `error` variant has a specific background color. For `<notification/>`, did you mean to add yellow background by default for the 'warning' variant?

Reply:
https://github.com/stellar/stellar-design-system/pull/258